modules/standalone-to-operator-backup-restore.adoc

```diff
@@ -0,0 +1,379 @@
+          secret_key: ##########
+          storage_path: /datastorage/registry
+----
+
+. Store the `access_key` and `secret_key` from the Quay `config.yaml` file in your `~/.aws` directory:
++
+[source,terminal]
+----
+$ touch ~/.aws/credentials
+----
+
+. Optional: Check that your `access_key` and `secret_key` are stored:
++
+[source,terminal]
+----
+$ cat > ~/.aws/credentials << EOF
+[default]
+aws_access_key_id = ACCESS_KEY_FROM_QUAY_CONFIG
+aws_secret_access_key = SECRET_KEY_FROM_QUAY_CONFIG
+EOF
+----
++
+Example output:
++
+[source,terminal]
+----
+aws_access_key_id = ACCESS_KEY_FROM_QUAY_CONFIG
+aws_secret_access_key = SECRET_KEY_FROM_QUAY_CONFIG
+----
++
+[NOTE]
+====
+If the `aws cli` does not automatically collect the `access_key` and `secret_key` from the ``~/.aws/credentials file`, you can, you can configure these by running `aws configure` and manually inputting the credentials.
+====
+
+. In your `quay-backup` directory, create a `bucket_backup` directory:
++
+[source,terminal]
+----
+$ mkdir /tmp/quay-backup/bucket-backup
+----
+
+. Backup all blobs from the S3 storage:
++
+[source,terminal]
+----
+$ aws s3 sync --no-verify-ssl --endpoint-url https://PUBLIC_S3_ENDPOINT:PORT s3://QUAY_BUCKET/ /tmp/quay-backup/bucket-backup/
+----
++
+[NOTE]
+====
+The `PUBLIC_S3_ENDPOINT` can be read from the Quay `config.yaml` file under `hostname` in the `DISTRIBUTED_STORAGE_CONFIG`. If the endpoint is insecure, use `http` instead of `https` in the endpoint URL.
+====
+
+Up to this point, you should have a complete backup of all Quay data, blobs, the database, and the `config.yaml` file stored locally. In the following section, you will migrate the standalone deployment backup to Red Hat Quay on OpenShift Container Platform.
+
+== Migrating a standalone {productname} deployment to OpenShift Container Platform.
+
+
+.Prerequisites
+
+* Your standalone {productname} data, blobs, database, and `config.yaml` have been backed up.
+* {productname} is deployed on OpenShift Container Platform using the Quay Operator.
+* A `QuayRegistry` with all components set to `managed`.
+
+.Procedure
+
+[NOTE]
+====
+The procedure in this documents uses the following namespace: `quay-enterprise`.
+====
+
+. Scale down the {productname} Operator:
++
+[source,terminal]
+----
+$ oc scale --replicas=0 deployment quay-operator.v3.6.2 -n openshift-operators
+----
+
+. Scale down the application and mirror deployments:
++
+[source,terminal]
+----
+$ oc scale --replicas=0 deployment QUAY_MAIN_APP_DEPLOYMENT QUAY_MIRROR_DEPLOYMENT
+----
+
+. Copy the database SQL backup to the Quay PostgreSQL database instance:
++
+[source,terminal]
+----
+$ oc cp /tmp/user/quay-backup/quay-database-backup.sql quay-enterprise/quayregistry-quay-database-54956cdd54-p7b2w:/var/lib/pgsql/data/userdata
+----
++
+[NOTE]
+====
+In versions of OpenShift Container Platform earlier than 4.10, you must use the following format: `$ oc cp <file-spec-src> <file-spec-dest> [flags]`.
+====
```

Comment:
I think we don't need to add this `NOTE`.  In my last comment, I mean the `oc cp` command is the same in OCP 4.6, 4.7, 4.8. and 4.10.  I guess the other `oc` versions should be the same.  So the below one command is enough. 
```
----
$ oc cp /tmp/user/quay-backup/quay-database-backup.sql <quay-namespace>/<quay_pod_name>:/var/lib/pgsql/data/userdata
----
```